lib/please_cli/please_cli/deploy.py

```diff
@@ -628,9 +628,16 @@ def cmd_DOCKERHUB(ctx,
         # Puth the hash to the end of the tag
         image_tag_versioned = '-'.join(reversed(tag_base.split('-', 1)))
         # Stable tag, e.g. shipit-api-staging
-        image_tag = docker_image_tag_format.format(project=project.replace('/', '-'),
-                                                   nix_path_attribute=nix_path_attribute.replace('/', '-'),
+        nix_path_attribute_ = nix_path_attribute.replace('/', '-')
+        project_ = project.replace('/', '-')
```

Comment:
@rail do you maybe remember why we normalize this? I'm leaning towards not changing this at all and using '_' as a separator. examples: `shipit/api_docker_staging_latest`

reasons:
- i would only normalize the names when we are forces to do it
- i think `_` is a better separator makes more sense since `-` can and is used in nix names and also in python distribution names.